src/contracts/core/RewardsCoordinatorStorage.sol

```diff
@@ -85,6 +85,8 @@ abstract contract RewardsCoordinatorStorage is IRewardsCoordinator {
     mapping(address => mapping(bytes32 => bool)) public isRewardsSubmissionForAllHash;
     /// @notice Mapping: address => bool to check if the address is permissioned to call createRewardsForAllSubmission
     mapping(address => bool) public isRewardsForAllSubmitter;
+    /// @notice Mapping: operator => avs => operatorSetId => OperatorCommissionUpdate history
+    mapping(address => mapping(address => mapping(uint32 => mapping(RewardType => OperatorCommissionUpdate[])))) public operatorCommissionUpdates;
```

Comment:
Might make sense to add a getter so external contracts can fetch this array's length to iterate over it.

Example:
```
function operatorCommissionUpdatesLength(operator, avs, operatorSetId) => length
```

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/627/files#diff-833651254f860f4b130eeea1ab4455ed94e4bc70117a7448f817689675b5e8f6R619-R626
Ah I actually added this in but wasn't included in the interface. Also for consistency lets keep functions out of the abstract storage contracts